packages/reference/src/components/ResourceEntityErrorCard/FunctionInvocationErrorCard.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { Flex, Text, TextLink } from '@contentful/f36-components';
+
+import { MissingEntityCard } from '..';
+
+import { ErrorCircleOutlineIcon, ExternalLinkIcon } from '@contentful/f36-icons';
+
+import { useResourceProvider } from '../../common/EntityStore';
+
+type FunctionInvocationErrorLogLinkProps = {
+  organizationId: string;
+  appDefinitionId: string;
+  functionId?: string;
+  status: 'error' | 'success' | 'loading';
+};
+
+function FunctionInvocationErrorLogLink(props: FunctionInvocationErrorLogLinkProps) {
+  const functionLink = `/account/organizations/${props.organizationId}/apps/definitions/${props.appDefinitionId}/functions/${props.functionId}/logs`;
+
+  const children = (
+    <Text fontColor="colorNegative">
+      &nbsp;For more information, go to&nbsp;
+      <TextLink icon={<ExternalLinkIcon />} alignIcon="end" href={functionLink}>
+        function logs
+      </TextLink>
+    </Text>
+  );
+
+  return (
+    <Flex justifyContent="left" alignItems="center">
+      <ErrorCircleOutlineIcon variant="negative" />
+      <Text fontColor="colorNegative">&nbsp;Function invocation error.</Text>
+      {props.status === 'success' && props.functionId && children}
```

Comment:
This is a potential edge case where we can't get the function id to successfully crate the function log link.